portal/app/farm_updater/std_api.py

```diff
@@ -17,34 +17,37 @@ def __init__(self, url=settings.SUN_TECH_DRIVE_URL, username=None, password=None
 
     def login(self):
         session = requests.Session()
-        session.post(self.url + self.LOGIN_ENDPOINT, data=self.credentials)
+        try:
+            session.post(self.url + self.LOGIN_ENDPOINT, data=self.credentials, verify=False)
+        except requests.exceptions.RequestException as exc:
```

Comment:
Why did you add this try except? All you do is print the exception but then you're still using the session?

Reply:
forgot to delete that...